modules/nf-core/dysgu/main.nf

```diff
@@ -0,0 +1,59 @@
+
+process DYSGU {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/dysgu:latest' :
+        'biocontainers/dysgu:latest' }"
+
+    input:
+
+    tuple val(meta), path(input_bam), path(input_bam_index)
+    tuple val(meta3), path(fasta), path(fai)
```

Comment:
```suggestion
    tuple val(meta), path(input_bam), path(input_bam_index)
    tuple val(meta2), path(fasta), path(fai)
```
why did you change from reference? I think that was quite descriptive

Reply:
Hi @famosab I changed it because when I was runing `nf-core modules lint dysgu` it was giving errors reference_fasta not found in meta.yml file, However now it's working.